pagopa/pn-frontend
# fix: differenziate tos acceptance page using mui-italia ToSAgreement …

…component

## Short description
Updates mui-italia in all packages and differenziate ToS agreement pages in personafisica and pa webapps

## List of changes proposed in this pull request
- Update mui-italia
- Change ToSAcceptance page in pn-pa-webapp, using mui-italia component and removing switch
- Change ToSAcceptance page in pn-personafisica-webapp, using mui-italia component

## How to test
ToSAcceptance page in pn-pa-webapp should not display the switch
ToSAcceptance page in pn-personafisica-webapp should display the switch
mui-italia package version should be 0.8.1 in all packages
